maxim/logger/mistral/client.py

```
@@ -0,0 +1,272 @@
+from typing import Any, List, Optional, Tuple
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)